snapshot/db.go

```diff
@@ -0,0 +1,50 @@
+package snapshot
+
+import (
+	"github.com/scootdev/scoot/snapshot/git/repo"
+)
+
+// ID identifies a value in DB. Opaque to the client.
```

Comment:
nit - capitalize Value (here and below). And does it make sense to add a breadcrumb to doc.go wrt Value?